Jenkinsfile

```diff
@@ -448,7 +449,7 @@ ansiColor('xterm') {
 
                   // Rebuild with correct version number
                   sh 'npm run build'
-                  sh 'npm run build:script'
+                  sh "npm run build:script -- --ver=${version}"
```

Comment:
is `version` here the same as the sdk package.json version?

Reply:
yes